test/util/server/server.go

```diff
@@ -386,7 +386,7 @@ func IsServerHealthy(url url.URL) (bool, error) {
 		},
 	})
 
-	url.Path = "/healthz"
+	url.Path = "/healthz/ready"
```

Comment:
Just needs to be evaluated on it's own as a change to semantics of this method (and maybe rename it), didn't want it to slip past as a side-effect of this pr